src/pkg/storage/s3.go

```diff
@@ -19,9 +21,12 @@ type S3Config struct {
 
 // config key consts
 const (
+	keyS3AccessKey      = "s3_accessKey"
```

Comment:
CamelCase here as well, please.
```suggestion
	keyS3AccessKey      = "s3_access_key"
```